2-mandatory/test.js

```diff
@@ -0,0 +1,72 @@
+/*const STOCKS = ["aapl", "msft", "amzn", "googl", "tsla"];
+
+const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
+  [179.19, 180.33, 176.28, 175.64, 172.99], // AAPL
+  [340.69, 342.45, 334.69, 333.2, 327.29], // MSFT
+  [3384.44, 3393.39, 3421.37, 3420.74, 3408.34], // AMZN
+  [2951.88, 2958.13, 2938.33, 2928.3, 2869.45], // GOOGL
+  [1101.3, 1093.94, 1067.0, 1008.87, 938.53], // TSLA
+];
+
+
+  console.log(stockPriceAverage);
+}
+getAveragePrices(CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS);
+
+/*function getPriceChanges(closingPricesForAllStocks) {
+  priceChangeArray = [];
```

Comment:
@lalehkoupa  did you comment out your solution?

Reply:
That was just an extra js page that I created to test some of my codes there and now I deleted it.